main.py

```diff
@@ -1,29 +1,30 @@
 import functions_framework
 from google.cloud import tasks_v2
-import os
+from utils import logger
+import config
 import json
 
-SERVICE_ACCOUNT_EMAIL = os.environ.get("SERVICE_ACCOUNT_EMAIL")
-PROJECT = os.environ.get("GCP_PROJECT_NAME")
-QUEUE = os.environ.get("GCP_QUEUE_NAME")
-LOCATION = os.environ.get("GCP_RESOURCE_LOCATION")
-TARGET_FUNCTION_URL = os.environ.get("TARGET_FUNCTION_URL")
 
 @functions_framework.http
 def handle(req):
-    payload = json.dumps(req.get_json())
-    client = tasks_v2.CloudTasksClient()
-    parent = client.queue_path(PROJECT, LOCATION, QUEUE)
-    task = {
-        "http_request": { # Specify the type of request.
-            "http_method": tasks_v2.HttpMethod.POST,
-            "url": TARGET_FUNCTION_URL, # The full url path that the task will be sent to.
-            "body": payload.encode('utf-8'),
-            "headers": {"Content-type": "application/json"}
+    try:
+        payload = json.dumps(req.get_json())
+        logger.info(f"Payload: {payload}")
+
+        client = tasks_v2.CloudTasksClient()
+        parent = client.queue_path(config.PROJECT, config.LOCATION, config.QUEUE)
+        task = {
+            "http_request": { # Specify the type of request.
+                "http_method": tasks_v2.HttpMethod.POST,
+                "url": config.TARGET_FUNCTION_URL, # The full url path that the task will be sent to.
```

Comment:
Let's remove the comment
```suggestion
                "url": config.TARGET_FUNCTION_URL,
```

Reply:
Sure, removing it.